en/components/financialchart.md

```diff
@@ -86,8 +86,8 @@ var data = [
 The following code demonstrates how to bind the financial chart to the above data. 
 
 ```html 
- <ig-financial-chart [dataSource]="data">
- </ig-financial-chart>
+ <igx-financial-chart [dataSource]="data" style="width:700px; height:500px">
```

Comment:
width="850px" height="600px"

Note, you should probably use at least **850**px width and **600**px height for finchart in all samples, because this will give enough room for the toolbar.